tests/foreman/api/test_multiple_paths.py

```diff
@@ -116,14 +117,16 @@ class EntityTestCase(APITestCase):
         entities.Architecture,
         entities.AuthSourceLDAP,
         entities.ComputeProfile,
-        entities.ComputeResource,
+        entities.AbstractComputeResource,
```

Comment:
I will update. I got this doubt when first thought to drop it. But I decided to leave to make sure it also works.